frontend/src/components/UserInfoCard.tsx

```diff
@@ -18,13 +18,31 @@ import { ModalTypes } from './modals/types';
 import { modalsActions } from '../redux/modals';
 import { api } from '../api/api';
 import { paths } from '../routers/paths';
+import { ExitButton } from './forms/SvgButton';
 
 const classes = classNames.bind(css);
 
 const ProjectRow: TableRow<RoadmapRole> = ({ item: roadmapRole, style }) => {
   const { t } = useTranslation();
+  const dispatch = useDispatch();
   const { roadmapId, type } = roadmapRole;
   const roadmap = useSelector(idRoadmapSelector(roadmapId));
+
+  const leaveProject = useCallback(
+    (event) => {
```

Comment:
```suggestion
    (event: SyntheticEvent) => {
```

For some reason eslint doesn't complain about "implicit any" when wrapped in `useCallback`